src/base/misc/fatfs.c

```diff
@@ -506,7 +506,7 @@ int read_boot(fatfs_t *f, unsigned char *b)
 
     build_boot_blk(f, b);
 
-    memcpy(b + 0x03, "DOSEMU10", 8);
+    memcpy(b + 0x03, "IBM  3.3", 8);
```

Comment:
Hmm. Just wondering, can we have that for
the affected DOSes only? Or maybe for any DOS
but freedos? What do you think?
Anyway, whatever you feel is better.